hudi-common/src/main/java/org/apache/hudi/common/fs/FSUtils.java

```diff
@@ -291,6 +292,63 @@ public static Map<String, List<StoragePathInfo>> getFilesInPartitions(HoodieEngi
     }
   }
 
+  /**
+   * Get all the files in the given partition path.
+   *
+   * @param storage Hoodie Storage
+   * @param partitionPathIncludeBasePath The full partition path including the base path
+   * @param filesNamesUnderThisPartition The names of the files under this partition for which file status is needed
+   * @param ignoreMissingFiles If true, missing files will be ignored and empty Option will be added to the result list
+   * @return List of file statuses for the files under this partition
+   */
+  public static List<Option<StoragePathInfo>> getPathInfoUnderPartition(HoodieStorage storage,
```

Comment:
If this should only be used by listing the log files, move this method to the corresponding class using this and make it private.

Reply:
I have moved the utility method to `RollbackHelperTableVersionSix`